/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 27 1 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.552274942398


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_27_1_50_4_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 26 1 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.537656068802


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_26_1_50_9_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 24 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.554216861725


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_24_2_50_9_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 22 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.53502702713


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_22_2_50_16_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 20 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.515538930893


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_20_2_50_25_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 18 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.518062829971


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_18_2_50_36_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 16 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.561244010925


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_16_2_50_49_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 14 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.658549070358


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_14_2_50_64_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 12 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.525439023972


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_12_2_50_81_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 10 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.59715294838


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_10_2_50_100_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
/home/djsaunde/anaconda2/lib/python2.7/site-packages/brian_no_units.py:4: UserWarning: Turning off units
  warnings.warn("Turning off units")
False


test pairs no_weight_dependence postpre 8 2 50 weight_sharing 8 0.0 0.0 10 False


time needed to load test set: 0.514167785645


...creating neuron group: A
...creating recurrent connections
Traceback (most recent call last):
  File "spiking_conv_patch_connectivity_MNIST.py", line 1378, in <module>
    build_network()
  File "spiking_conv_patch_connectivity_MNIST.py", line 751, in build_network
    weight_matrix = get_matrix_from_file(weight_path + conn_name + '_' + ending + '.npy', conv_features * n_e, conv_features * n_e)
  File "spiking_conv_patch_connectivity_MNIST.py", line 97, in get_matrix_from_file
    readout = np.load(file_name)
  File "/home/djsaunde/anaconda2/lib/python2.7/site-packages/numpy/lib/npyio.py", line 362, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '../weights/conv_patch_connectivity_weights/AeAe_pairs_8_2_50_121_no_weight_dependence_postpre_weight_sharing_8_0.0.npy'
